Makefile

```diff
@@ -5,8 +5,13 @@ default: build
 build:
 	go build -o mysql-tester ./src
 
-test:
-	go test -cover ./...
+debug:
+	go build -gcflags="all=-N -l" -o mysql-tester ./src
+
+test: build
+	# waiting on pr/46
```

Comment:
Looks like https://github.com/pingcap/mysql-tester/pull/46 needs a rebase and new reviewers as well.

Reply:
This part actually works now.